package.json

```diff
@@ -40,8 +40,7 @@
         "eslint": "^6.7.2",
         "@fluent/bundle": "^0.14.0",
         "@fluent/syntax": "^0.14.0",
-        "gh-pages": "^2.1.1",
-        "gitbook-cli": "^2.3.2",
+        "gh-pages": "^6.1.1",
```

Comment:
Should probably drop this and the `npm run deploy` script, if/as we're going to presume that the action will take care of that.

Reply:
Removed this and the deploy action, but I didn't run any npm command (as said, I'd prefer for someone who understands JS to do that)